static/app/utils/onDemandMetrics/index.tsx

```diff
@@ -58,3 +72,14 @@ export const extrapolatedAreaStyle = {
   },
   opacity: 1.0,
 };
+
+export function hasOnDemandMetricAlertFeature(organization) {
```

Comment:
nit: maybe just add type for the organization here

Reply:
oh, right. Was not aware we even allow implicit any 😮